extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSBatchScan.scala

```diff
@@ -65,6 +65,16 @@ class TPCDSBatchScan(
     val chuck = partition.asInstanceOf[TPCDSTableChuck]
     new TPCDSPartitionReader(chuck.table, chuck.scale, chuck.parallelism, chuck.index, schema)
   }
+
+  override def estimateStatistics(): Statistics = {
+    new Statistics {
+      override def sizeInBytes(): OptionalLong = {
+        OptionalLong.of(rowCount * schema.defaultSize)
```

Comment:
The TPC-DS document actually provides a table with estimate row size of each table. I'm now sure which one is better here.
https://tpc.org/TPC_Documents_Current_Versions/pdf/TPC-DS_v3.2.0.pdf
<img width="796" alt="image" src="https://user-images.githubusercontent.com/26535726/167622496-2dbc1017-668c-4775-8eba-7bcd03640df2.png">